.github/workflows/deploy-prod&demo.yaml

```diff
@@ -0,0 +1,60 @@
+name: Deploy to production & demo
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
```

Comment:
Skal det være noe i `workflow_dispatch`?

Reply:
Trengs ikke, har den kun med for å kunne manuelt trigge workflowen fra GitHub, da vi nå og da kan ha behov for det.
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch